pkg/kafkaClient/reader.go

```diff
@@ -24,13 +24,19 @@ func StartTopicReader(ctx context.Context, messageChannel chan string, errorChan
 	// initialize a new reader with the brokers and topic
 	// the groupID identifies the consumer and prevents
 	// it from receiving duplicate messages
-
+	r := kafka.NewReader(kafka.ReaderConfig{
+		Brokers: brokerURLs,
+		Topic:   inbound,
+		GroupID: groupID,
+	})
 
 	brokerStr := "Brokers: "
 	for i := 0; i < len(brokerURLs); i++ {
 		brokerStr += brokerURLs[i]
 	}
-	log.Printf("StartTopicReader(%s,%s,%s)", brokerStr, inbound, groupID)
+	if logger.IfInfo() {
+		logger.Info("StartTopicReader(%s,%s,%s)", brokerStr, inbound, groupID)
```

Comment:
@donNewtonAlpha - it's probably one for another PR, but I think you should use our own logging framework in `onos-lib-go` like all the other modules do